### PR TITLE
chore(material-ui): Upgrade from material-ui to mui

### PR DESCRIPTION
**Pull request by fortunato**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,20 +57,19 @@
...
-		"@testing-library/user-event": "^13.1.9",
-		"@types/jest": "^26.0.23",
-		"@types/material-ui": "^0.21.8",
-		"@types/react-dom": "^17.0.5",
-		"@types/react-select": "^5.0.1",
-		"@types/styled-components": "^5.1.9",
+		"@storybook/addon-actions": "^6.5.9",
+		"@storybook/addon-docs": "^6.5.9",
+		"@storybook/addon-essentials": "^6.5.9",
+		"@storybook/addon-links": "^6.5.9",
```
Comment (dlnr) [inline package.json:63]: De adon @storybook/addon-a11y en @storybook/addon-interactions zijn dingen waar ik graag meer mee wil doen, maar dat is voor later misschien. Vooral accessibility is vrij laagdrempelig. Of heb je een reden om die niet te gebruiken?

File: package.json [hunk 2]
```diff
@@ -95,29 +94,31 @@
...
 	},
 	"dependencies": {
-		"@amsterdam/asc-assets": "^0.29.3",
-		"@amsterdam/asc-ui": "^0.31.0",
-		"@material-ui/core": "^4.11.4",
-		"@material-ui/data-grid": "^4.0.0-alpha.37",
-		"@material-ui/icons": "^4.11.2",
+		"@amsterdam/asc-assets": "^0.36.0",
+		"@amsterdam/asc-ui": "^0.36.0",
+		"@emotion/styled": "^11.9.3",
```
[the remark this answers is not in the document]
Comment (fortunato) [reply package.json]: You're totally right :ok_hand:

File: src/common/src/DocumentTable/ColumnFilterStyle.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import { Input, themeColor } from '@amsterdam/asc-ui';
-import CancelIcon from '@material-ui/icons/Cancel';
+import CancelIcon from '@mui/icons-material/Cancel';
```
[the remark this answers is not in the document]
Comment (fortunato) [reply src/common/src/DocumentTable/ColumnFilterStyle.ts:3]: If the icon set would be rich enough, yes. In this case a design was created with an alternate icon that we sourced from MUI. Typically we don't have resources available for creating new icons.

File: package.json (hunk 1 above)
Comment (fortunato) [reply package.json:63]: I'm totally happy to introduce these. But that's out of scope for this PR IMO

File: src/common/src/EditableInput/EditableInputStyles.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import styled from 'styled-components';
 import { Input } from '@amsterdam/asc-ui';
 
-import CancelIcon from '@material-ui/icons/Cancel';
-import ReplayIcon from '@material-ui/icons/Replay';
+import CancelIcon from '@mui/icons-material/Cancel';
+import ReplayIcon from '@mui/icons-material/Replay';
```
[the remark this answers is not in the document]
Comment (fortunato) [reply src/common/src/EditableInput/EditableInputStyles.ts:5]: Fair point. I will however not refactor this component (which ended up not getting used anywhere in the end at least for the moment). I'd argue this is out of scope for this PR.

**Pull request merged**